WARNING:simsopt.objectives.least_squares:Function evaluation failed for <bound method Vmec.aspect of Vmec1 (nfp=2 mpol=5 ntor=5)>
WARNING:simsopt.objectives.least_squares:Function evaluation failed for <bound method Vmec.aspect of Vmec1 (nfp=2 mpol=5 ntor=5)>
WARNING:simsopt.objectives.least_squares:Function evaluation failed for <bound method Vmec.aspect of Vmec1 (nfp=2 mpol=5 ntor=5)>
WARNING:simsopt.objectives.least_squares:Function evaluation failed for <bound method Vmec.aspect of Vmec1 (nfp=2 mpol=5 ntor=5)>
/marconi/home/userexternal/rjorge00/simsopt/src/simsopt/mhd/vmec_diagnostics.py:237: RuntimeWarning: invalid value encountered in sqrt
  residuals3d[js, :, :] = np.sqrt(self.weights[js] * nfp * dtheta * dphi / V_prime[js] * sqrtg[js, :, :]) \
WARNING:simsopt.objectives.least_squares:Function evaluation failed for <bound method Vmec.aspect of Vmec1 (nfp=2 mpol=5 ntor=5)>
WARNING:simsopt.objectives.least_squares:Function evaluation failed for <bound method Vmec.aspect of Vmec1 (nfp=2 mpol=5 ntor=5)>
WARNING:simsopt.objectives.least_squares:Function evaluation failed for <bound method Vmec.aspect of Vmec1 (nfp=2 mpol=5 ntor=5)>
Traceback (most recent call last):
  File "/marconi_scratch/userexternal/rjorge00/single_stage_optimization/./main.py", line 91, in <module>
    vmec_input_filename = vmec_input_files[-1]
IndexError: list index out of range
Traceback (most recent call last):
  File "/marconi_scratch/userexternal/rjorge00/single_stage_optimization/./main.py", line 91, in <module>
    vmec_input_filename = vmec_input_files[-1]
IndexError: list index out of range
slurmstepd: error: *** JOB 10861767 ON r134c07s03 CANCELLED AT 2023-01-30T07:06:01 DUE TO TIME LIMIT ***
